src/runtime/linux.rs

```diff
@@ -828,13 +828,13 @@ impl TryFrom<&str> for LinuxNamespaceType {
 
     fn try_from(namespace: &str) -> Result<Self, Self::Error> {
```

Comment:
📝 We use it in here:
https://github.com/containers/youki/blob/d6c59966c001a862949db708043863aededfc08b/crates/libcontainer/src/container/tenant_builder.rs#L478

Reply:
Yeah, I use the same way `LinuxNamespace::try_from(ns_str)?` in my project too.
Thx @utam0k @saschagrunert and **`the great project oci-spec-rs`**